packages/tabs/src/styled/StyledTabs.ts

```diff
@@ -5,78 +5,26 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import styled from 'styled-components';
 import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
-import { StyledTab } from './StyledTab';
-import { StyledTabPanel } from './StyledTabPanel';
-import { StyledTabList } from './StyledTabList';
 
 const COMPONENT_ID = 'tabs.tabs';
 
 interface IStyledTabsProps {
-  /**
-   * Displays vertical TabList styling
-   */
   isVertical?: boolean;
 }
 
-const verticalStyling = ({ theme }: ThemeProps<DefaultTheme>) => {
-  return css`
-    display: table;
-
-    ${StyledTabList} {
-      display: table-cell;
-      margin-bottom: 0;
-      border-bottom: none;
-      vertical-align: top;
-    }
-
-    ${StyledTab} {
-      display: block;
-      margin-bottom: ${theme.space.base * 5}px;
-      margin-left: ${theme.rtl && '0'};
-      border-left: ${theme.rtl && '0'};
-      border-bottom-style: none;
-      /* stylelint-disable property-case, property-no-unknown */
-      border-${theme.rtl ? 'right' : 'left'}-style: ${theme.borderStyles.solid};
-      border-${theme.rtl ? 'right' : 'left'}-color: transparent;
-      /* stylelint-enable property-case, property-no-unknown */
-      padding: ${theme.space.base}px ${theme.space.base * 2}px;
-      text-align: ${theme.rtl ? 'right' : 'left'};
-
-      &:last-of-type {
-        margin-bottom: 0;
-      }
-
-      &:focus-visible::before,
-      &[data-garden-focus-visible]::before {
-        top: ${theme.space.base}px;
-        right: ${theme.space.base}px;
-        left: ${theme.space.base}px;
-      }
-    }
-
-    ${StyledTabPanel} {
-      /* stylelint-disable-next-line property-no-unknown */
-      margin-${theme.rtl ? 'right' : 'left'}: ${theme.space.base * 8}px;
-      vertical-align: top;
-    }
-  `;
-};
-
 /**
  * Accepts all `<div>` props
  */
 export const StyledTabs = styled.div.attrs<IStyledTabsProps>({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION
 })<IStyledTabsProps>`
-  display: block;
+  display: ${props => (props.isVertical ? 'table' : 'block')};
```

Comment:
I'm wondeirng if using flex might be preferable for handling flow. 🤔 `display: table` can impact the element's role for some screen readers ([source](https://www.456bereastreet.com/archive/201110/using_displaytable_has_semantic_effects_in_some_screen_readers/)). Granted, it's a small subset of SRs, so maybe not a huge deal?

Reply:
I also confirmed using a fake table demo in the link above that Chrome + JAWS treats CSS tables as actual tables (so long as they have more than one cell). 😬

Reply:
@geotrev all valid concerns for sure. For this PR I want to retain exact parity with the previous styling, since that's already been through a11y audit.

> trying to use arrow keys backward through tabs requires one extra arrow key press to loop from the first to last tab

Looks like that might be related to the inclusion of a `disabled` tab in the Storybook example. Again, let's plan to address under a separate effort.